### PR TITLE
[Components][Form] apply some fixes to the Form events chapter

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |
- remove serial commas
- replace wrong event constant
- replace reference to form events overview that did not provide any
  context with a more meaningful `seealso` section

### CODE REVIEW

File: components/form/form_events.rst [hunk 1]
```diff
@@ -69,6 +67,11 @@ Normalized data  ``null``
 View data        ``null``
 ===============  ========
 
+.. seealso::
+
+    See all form at a glance in the
```
Comment (WouterJ) [inline components/form/form_events.rst]: What do you mean with "all form"? Did you mean "all form events"?

File: components/form/form_events.rst [hunk 2]
```diff
@@ -69,6 +67,11 @@ Normalized data  ``null``
...
+    :ref:`Form Events Information Table<component-form-event-table>`.
```
Comment (WouterJ) [inline components/form/form_events.rst]: there have to be a space before `<`

Comment (xabbuh) [inline components/form/form_events.rst]: fixed

File: components/form/form_events.rst (hunk 1 above)
Comment (xabbuh) [inline components/form/form_events.rst]: oops, yes, added the missing "events"

**Comment by weaverryan** [comment]
:thumbsup:

**Pull request merged**